file-formats/sajc/type/zif_aff_sajc_v1.intf.abap

```diff
@@ -34,6 +38,9 @@ INTERFACE zif_aff_sajc_v1
       "! Name of the class which contains the execute-method to run within the job
       "! $required
       class_name            TYPE zif_aff_types_v1=>ty_object_name_30,
+      "! <p class="shorttext">Executed Report</p>
+      "! Name of the report which is executed within the job
+      report_name           TYPE ty_report_name,
```

Comment:
I just came across "report". Can we change this to "program"? As far as I can see in SAPterm. "report" is the obsolete term for "executable program".
```suggestion
      "! <p class="shorttext">Executed Program</p>
      "! Name of the program which is executed within the job
      program_name           TYPE ty_report_name,
```